src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqScoreSpark.java

```diff
@@ -0,0 +1,134 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.ArgumentCollection;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.engine.spark.datasources.ReadsSparkSink;
+import org.broadinstitute.hellbender.engine.spark.datasources.ReadsSparkSource;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadsWriteFormat;
+import scala.Tuple2;
+
+import java.io.IOException;
+
+/**
+ * This Spark tool classifies reads taxonomically, assigning each taxon abundance scores. This is the last
+ *   step of the PathSeq pipeline for abundance quantification. The output is a tab-delimited table with the
+ *   following columns:
+ * <p>
+ * 1) taxonomic id
+ * 2) taxonomic classification
+ * 3) taxonomic type (order, family, genus, etc.)
+ * 4) name
+ * 5) raw abundance score:
+ * <p>
+ * score of species S = sum_{read r in reads aligned to S} 1 / ({# of species that r aligned to} x {length of species reference in Mbp})
+ * <p>
+ * score of higher taxa = sum of scores of child nodes
+ * <p>
+ * 6) abundance score normalized as a percentage of total pathogen-mapped reads
+ * 7) total number of reads aligned to this taxon
+ * 8) number of reads assigned uniquely to this taxon (i.e. mapped only to the node and/or its children)
+ * 9) total taxon reference sequence length
+ * <p>
+ * For unpaired reads, a particular alignment is considered a hit if it aligned with minimal coverage score (read length
+ *   less padding) given as a fraction of the full read length, and identity score (matches less deletions) given as a
+ *   fraction of the read with padded bases removed.
+ * For paired-end reads, an alignment to a particular taxon is considered a hit only if both reads aligned to that
+ *   taxon with minimal coverage and identity.
+ */
+
+@CommandLineProgramProperties(summary = "Performs taxonomic classification of BAM file reads that have been aligned to a reference of pathogens",
```

Comment:
"BAM file" seems superfluous in the comment unless you want to expand on it (ie mention the two different BAM inputs)